### DISCUSSION TITLE
Enable common Emoj´s for even better notes

### DISCUSSION THREAD

**Discussion by CobriMediaJulien** [TriliumNext/Trilium#4500] [Ideas] [answered]
Hallo Guys,

At first, thank you Zadam for this application. It is awesome. I have a simple question. I saw  in the dev tools that UTF 8 charset is enabled. But I´m not getting common Emojis or Unicode Emojis  to work. I mean emojis like these:

https://www.w3schools.com/html/html_emojis.asp

They don't work if I write the code with &#.
➡️ this arrow on the other hand, does work. Is there a way to add insert common emojis  to this bar?

![image](https://github.com/zadam/trilium/assets/56324041/6b703cdf-de8b-4418-b365-592630ce57dc)

For me, this would be a gamechanger to personalize my notes. What do you guys think?

Greetings to all!

**Comment by CobriMediaJulien** [comment, marked as answer]
I have to answer myself. I just looked into the CKEditor Docu and if u press [Windowskey] + [.] u open a emote and special character menu for all who didnt knew. For mac:  Ctrl+Cmd+Space  and for Linux i dont know.

@zadam if u want i canwrite an example for the trillium doku sheet that u can add. Greetingd and ty again

**Comment by purashy** [comment]
> for Linux i dont know.

For linux you can get software that is the exact as `CTRL + .` on windows. All depends on the flavor you are using. For KDE I think I used a software called `KDE Emoji Selector` or something.

To extend this answer about the actual notes and their emojis, you can do this by important some CSS.

I haven't found the perfect solution yet for this, but what I have right now is the following:

Create a note with the type `CSS`. on its content, write:
`@import "https://emoji-css.afeld.me/emoji.css";`

On its Owned Attributes, write:
`#appCss`

Press `CTRL + R` (this will reload trilium notes).

Now all you do is go to any note you have, go to its Owned Atributes, and write the icon you want. For example:

`#iconClass="em em-house"`

[You can find more icons here.](https://emoji-css.afeld.me)

**Reply by bonnepioche** [reply]
Hello,
Running KDE Neon, you can simply press CTRL + SHIFT + u <release>  and then the number of the unicode you want to display then enter.
Pressing CTRL+SHIFT gives you an "undernlined u" ![image](https://github.com/zadam/trilium/assets/16622600/0c863e69-ea72-41f7-93d0-6811ca23256b)
Ex: CTRL+SHIFT+u and then 12484 then ENTER.
Hope this helps,
